packages/grafana-data/src/events/pageBanner.ts

```diff
@@ -0,0 +1,27 @@
+import { BusEventWithPayload } from './types';
+
+/** @alpha */
+export class PageBannerDisplayEvent extends BusEventWithPayload<PageBannerEventPayload> {
+  static type = 'page-banner-display';
+}
+
+/** @alpha */
+export type PageBannerEventPayload = {
+  text: string;
```

Comment:
if we change this to string | React.Node cloud can put anything here (for example text with links)